packages/protocol/migrations/09_stabletoken.ts

```diff
@@ -60,10 +60,12 @@ module.exports = deploymentForCoreContract<StableTokenInstance>(
     if (goldPrice) {
       const fromAddress = truffle.networks[networkName].from
       const isOracle = config.stableToken.oracles.some((o) => eqAddress(o, fromAddress))
-      console.assert(
-        isOracle,
-        `Gold price specified in migration but ${fromAddress} not authorized as oracle`
-      )
+      if (!isOracle) {
+        console.warning(
+          `Gold price specified in migration but ${fromAddress} not explicitly authorized as oracle, authorizing...`
+        )
+        await sortedOracles.addOracle(stableToken.address, ensureLeading0x(fromAddress))
```

Comment:
are you doing this dynamically just so the contract deployer address doesn't have to be duplicated in both the the truffle config and migration config?

Reply:
Yeah I saw the migrations fail b/c the "from" address wasn't explicitly authorized, which is a pain. It's easier to whitelist the oracle now and remove it later via governance.